.npmignore

```diff
@@ -0,0 +1 @@
+*.tsbuildinfo
```

Comment:
question: i forget how .npmignore works. is it additive to what's in .gitignore?

Reply:
npm does not consider `.gitignore` when publishing files (otherwise build artifacts would not get published); it only considers the `files` prop (which is an allow-list) and the `.npmignore` file (which is a deny-list).